src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,19 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = 'Under Construction';
```

Comment:
- [ ]The initial state for the categories slice should be an empty array as you can see in the picture below. Please implement it. 
![image](https://github.com/Rafiduzzaman/bookstore/assets/91486791/657c165d-d942-4e44-bef4-1b2f103862f6)